README.md

```diff
@@ -2,79 +2,95 @@
+| node js / [passport.js](http://passportjs.org) | [passport-verify](https://github.com/alphagov/passport-verify) |
 
-```
-VERIFY_ENVIRONMENT            # The environment of the Verify Hub to run against - PRODUCTION, INTEGRATION, or COMPLIANCE_TOOL
+See [the API reference][vsp-api] for full details of the API.
 
-SERVICE_ENTITY_IDS            # A JSON string array containing the entity id of the service using Verify Service Provider, e.g. '["http://entity-id"]'
-                              # If you have multiple services using a single Verify Service Provider you should provide all of their entity IDs in this array.
-MSA_ENTITY_ID                 # The SAML Entity Id that identifies the Relying Party's Matching Service Adapter
-MSA_METADATA_URL              # The URL to the Matching Service Adapter's SAML metadata.
+## Run
 
-SAML_SIGNING_KEY              # A base64 encoded RSA private key that is used for signing the request to Verify
-SAML_PRIMARY_ENCRYPTION_KEY   # A primary base64 encoded PKCS8 RSA private key that is used to decrypt encrypted SAML Assertions (see "Generating keys for testing")
-SAML_SECONDARY_ENCRYPTION_KEY # (Optional - default empty) A secondary base64 encoded PKCS8 RSA private key that is used to decrypt encrypted SAML Assertions that
-                              # will be used during certificate rotation events (see "Generating keys for testing")
+You can run the the VSP using several commands, depending on your development needs:
 
-PORT                          # (Optional - default 50400) The TCP port where the application will listen for HTTP traffic
-LOG_LEVEL                     # (Optional - default INFO) The threshold level for logs to be written (e.g. DEBUG, INFO, WARN, or ERROR)
-```
+| Command       | Use case                               |
+| ------------- | ----------------------------------------- |
+| `development` | Local development of a VSP client <br> The VSP is connected to a test tool hosted by the GOV.UK Verify team. |
+| `server`      | End-to-end testing and running in production <br> The VSP uses the configuration provided to connect to <br> a full-scale of the GOV.UK Verify Hub.                                          |
+| `check`       | Validates your configuration file.        |
 
-As Verify Service Provider is a Dropwizard application, you can also configure it with all [options provided by Dropwizard](http://www.dropwizard.io/1.3.5/docs/manual/configuration.html).
+### `development`
 
-### Generate keys for testing
+You can use `development` command if you're building your own client for the VSP.The command starts the VSP connected to a test tool hosted by the GOV.UK Verify team. The test tool acts as a placeholder for the GOV.UK Verify Hub. This means you can use your local setup to test if your service can respond appropriately to all possible scenarios in a Verify journey.
```

Comment:
You can use the?

Reply:
😅

Reply:
__WILDCARD__: Maybe we want to call it test service rather than tool as tool to me implies that you are running something locally.

Reply:
I like the idea. 'Test tool' evolved from 'Compliance Tool' anyway, not much thought went into the right way to call it.

How about 'testing service'?
Does a service have commands or modes? 😏